### DISCUSSION TITLE
Service Account Users are really difficult to reach from UI perspective

### DISCUSSION THREAD

**Discussion by dmarrazzo** [keycloak/keycloak#21513] [Feedback]
I'm a newbie of keycloak maybe for such a reason, I found pretty difficult to understand that:

1. Behind a client (service account enabled) there is a user
2. This user can be configured only from "Service Account Roles" tab
3. I have to hover the pointer over the name to be teleported in the **hidden** user configuration
    [Screencast](https://github.com/keycloak/keycloak/assets/2084600/a01bf088-8135-48dd-8f50-ec5ec2a4dccc)

I believe that it would be much easier if the user view has a flag to include the service account users.
However, I strongly believe that this requires some love from a UI designer...

Thank you

**Comment by ssilvert** [comment]
Hi @dmarrazzo.  You are looking at the old UI.  If you bring up the new admin console you will see a friendly info message and a link.  It could probably still be improved some more but I think it's a lot better than the old one.  No more hovering.

<img width="502" alt="ServiceAccountUserLink" src="https://github.com/keycloak/keycloak/assets/507202/bd241c21-0fc3-4d06-b5cc-2a0c92c2736e">
